src/com/TTT/Main.java

```diff
@@ -7,16 +7,16 @@ public static void main(String[] args) {
     }
 
     private static void run() {
-        BoardSizeUI optionsUI = new BoardSizeUI(System.out, System.in);
+        GameUI gameUI = new GameUI(System.out, new BoardPrinter(System.out));
+        BoardSizeUI optionsUI = new BoardSizeUI(System.out, System.in, gameUI);
         Board board = new Board(optionsUI.boardSize());
-        PlayerOptionUI players = new PlayerOptionUI(System.out, System.in);
+        PlayerOptionUI players = new PlayerOptionUI(System.out, System.in, gameUI);
```

Comment:
I have rather a lot of setup now. It isn't clear from this method where the behaviour around getting player options and creating objects ends. I could make board size UI return a board, but I wanted advice on how best to test a config/setup like this.